### ISSUE TITLE
Rework news page

### ISSUE THREAD

**Issue by DrDrij** [QuantEcon/website#138]
- [ ] dates down the LHS column
- [ ] three types of "posts" including news, article, workshop

**Comment by DrDrij** [comment]
I have added example labels on news items, see below. I will comment them out though in the Markdown so that we can push updates to live and get to labelling in the next round.

![image](https://github.com/user-attachments/assets/6353c2d7-8226-467b-b700-6d81d616de0b)

**Comment by DrDrij** [comment]
Note: the labels are auto generated from "type" in the front-matter:

```
---
layout: post
title: "Africa summer course 2024"
author: Matt McKay
excerpt: QuantEcon has just delivered our Africa summer course
#type: news
---
```

**Comment by mmcky** [comment]
thanks @DrDrij. Is there a reason for the # in type? Just learning the syntax. 

also I think it might be nice to do different colours for each type? What do you think?

**Comment by DrDrij** [comment]
Yes can do. It’s just commented out as they are examples - in case we
wanted to update the live site before doing.

If you could make a list of labels and colours I will add the logic.

**Comment by mmcky** [comment]
Let's leave them black for now and I will make a list of colours. 

Can we change `Workshops` to `workshops` for all lower case?

**Comment by DrDrij** [comment]
Yep you can put anything after type, and '' if it has a space. 

Gee I should have named that key label or badge rather than type so it's clear. I might do that.

**Comment by mmcky** [comment]
- [ ] update news page for tags `workshop`, `news`, and `article`

**Comment by DrDrij** [comment]
I have added a hash to comment out the tag for the latest news item so that no tags will show. 

![Image](https://github.com/user-attachments/assets/9b6c50e9-7015-4eb3-b37c-c12af76e1e6e)

@mmcky You've listed 3 types of tags. Would you like me to go through and see if I can apply them to some of the recent posts?
